integration_test/support/migration.exs

```diff
@@ -67,19 +67,13 @@ defmodule Ecto.Integration.Migration do
     false = exists? table(:users)
 
     create table(:users) do
-      add :name, :string
-      add :to_be_removed, :string
-    end
-
-    true = exists? table(:users)
-
-    alter table(:users) do
-      modify :name, :text
+      add :name, :text
       add :custom_id, :uuid
-      remove :to_be_removed
       timestamps
     end
 
+    true = exists? table(:users)
```

Comment:
Let's move this whole thing to the new file. We can keep now the `migration.exs` just the straight-up table definition and let's add all the migration logic checking to the new one. This way everything becomes clearer, wdyt? :D